src/libs/Network/index.js

```diff
@@ -236,27 +246,41 @@ function processNetworkRequestQueue() {
         processRequest(queuedRequest)
             .then(response => onResponse(queuedRequest, response))
             .catch((error) => {
+                // Cancelled requests should not be retried
+                if (error.name === CONST.ERROR.REQUEST_CANCELLED) {
+                    onError(queuedRequest, error);
+                    return;
+                }
```

Comment:
NAB: Also I wonder if we can / should handle this condition in `processRequest` 🤔

Reply:
Yes, but next PR please. And you can give me some ideas on how to do it if you have them 😄